plugins/cameraserver/src/main/java/edu/wpi/first/shuffleboard/plugin/cameraserver/widget/CameraServerWidget.java

```diff
@@ -161,9 +162,9 @@ private void applySettings() {
     if (getSource() instanceof CameraServerSource) {
       CameraServerSource source = (CameraServerSource) getSource();
       int compression = (int) compressionSlider.getValue();
-      int fps = frameRateField.getNumber();
-      int width = this.width.getNumber();
-      int height = this.height.getNumber();
+      int fps = ((Number) frameRateField.getNumber()).intValue();
+      int width = ((Number) this.width.getNumber()).intValue();
+      int height = ((Number) this.height.getNumber()).intValue();
```

Comment:
Why is this now necessary?
The types haven't changed, have they?

Reply:
Gson deserializes numbers are their own special type, which are then loaded into these properties (which cannot be checked at runtime due to erasure). Controls on a camera widget that was in a save file cannot be used because Java's trying to unbox an `Integer` into an `int`, but the runtime type may be a `LazilyParsedNumber` instead

Reply:
Derp. Wow. But why are you casing the type of `this.height.getNumber` which is of type `Number` to `Number`?

Reply:
It's of type `Integer` (compile time) but can possibly be a different subclass of `Number` at runtime

Reply:
Reverted this change, since the underlying issue was fixed in #572